### PR TITLE
✨ Add how to play page

### LINKED ISSUE #16: Add how to play page
- [x] Create the how to play page out of components and images

### PR DESCRIPTION
**Pull request by cboenke**
Closes #16 

- Game control part will be added later
- CSS needs final styling after that

### CODE REVIEW

**Review by mrana17 — APPROVED** [review]
lgtm 🚀

**Review by DNaunin — APPROVED** [review]
Good Job! 💍

File: pages/how-to-play.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import Head from "next/head";
+import Button from "../components/button/Button";
+import Introduction from "../components/introduction/Introduction";
+import styles from "../styles/HowToPlay.module.css";
+
+const play = {
+  imgSrc: "/playSymbol.svg",
+  href: "#",
```
Comment (phlgr) [inline pages/how-to-play.tsx]: I would define this in the button

File: pages/how-to-play.tsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    <div className={styles.container}>
+      <Head>
+        <title>Placeholder for game name</title>
+        <link rel="icon" href="/favicon.ico" />
+      </Head>
+      <main className={styles.main}>
+        <img src="/howToPlay.png" className={styles.img} />
+        <Introduction />
+        <div className={styles.button}>
+          <Button {...play} />
```
Comment (phlgr) [inline pages/how-to-play.tsx]:
```suggestion
          <Button 
              imgSrc="/playSymbol.svg"
              href="#"
           />
```

File: pages/index.tsx [hunk 3]
```diff
@@ -5,7 +5,7 @@ import styles from "../styles/Welcome.module.css";
 
 const intro = {
   imgSrc: "/iSymbol.svg",
-  href: "#",
+  href: "/how-to-play",
 };
```
Comment (phlgr) [inline pages/index.tsx]: Same here

File: styles/HowToPlay.module.css [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+}
+
+.img {
+  max-width: 110%;
+  display: grid;
+  justify-self: center;
+}
+
+.button {
+  justify-self: flex-end;
```
Comment (phlgr) [inline styles/HowToPlay.module.css]: Does this do anything? This does not seem to be placed inside a flex container

**Review by phlgr — CHANGES_REQUESTED** [review]

File: components/introduction/Introduction.module.css [hunk 5]
```diff
@@ -11,4 +11,5 @@
 
 .subheading {
```
Comment (RonjaMeinecke) [inline components/introduction/Introduction.module.css:12]:
```suggestion
.subHeading {
```

**Review by RonjaMeinecke — APPROVED** [review]
💅🏼looks good to me

**Pull request merged**